src/App.js

```diff
@@ -1,12 +1,229 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Counter from "./component/Counter";
+import Button from "./component/Button";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    //FRUIT BASKET COUNTER STATE
+    const [strawberry, setStrawberry] = useState(0)
+    const [banana, setBanana] = useState(0)
+    const [apple, setApple] = useState(0)
+    const [kiwi, setKiwi] = useState(0)
+
+    //COUNTER RESET BUTTON
+    function reset() {
+        setStrawberry(0);
+        setBanana(0);
+        setApple(0);
+        setKiwi(0);
+    }
+
+    //ORDER FORM STATE
+    const [state, setState] = useState({
+        name: "",
+        surname: "",
+        age: 0,
+        zip: "",
+        message: "",
+    })
+
+    //SET STATE FOR NAME, SURNAME, AGE, ZIP and MESSAGE IN ORDER FORM
+    function handleChange(e) {
+        const value = e.target.value;
+        setState({...state, [e.target.name]: value});
+    }
+
+    //STATE FOR FREQUENCY, TERMS AND SUBMIT BUTTON
+    const [delivery_time, setDelivery_time] = useState(false)
```

Comment:
Ik zou geen underscore gebruiken, maar gewoon in camelCase schrijven: `deliveryTime`